scripts/update_location_summaries.ts

```diff
@@ -6,6 +6,7 @@ import {
   fetchAllStateProjections,
   fetchAllMetroProjections,
   fetchAllCountyProjections,
+  fetchProjectionsRegion,
```

Comment:
_nit_: It looks like the following imports are not needed anymore

```ts
import { DatasetId } from '../src/common/models/Projection';
import { assert } from '../src/common/utils';
```

also the `INDIGENOUS_FIPS` list is not used anymore